scrappers/spiders/ELCIUDADANO_spider.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+import scrapy
+from scrapy.http.response.html import HtmlResponse
+
+logging.basicConfig(filename="run.log", encoding="utf-8", level=logging.DEBUG)
+
+
+def get_title(response: HtmlResponse) -> str:
+    return response.xpath('//*[contains(@class, "mb-4 the_title")]/text()').get()
+
+
+def get_epigraph(response: HtmlResponse) -> str:
+    return response.xpath('//*[contains(@class, "epigrafe mb-2")]/text()').get()
+
+
+def get_date(response: HtmlResponse) -> str:
+    try:
+        date = (
+            response.xpath('//span[contains(@class, "time-ago time-now")]')
+            .get()
+            .replace('<span class="time-ago time-now" data-date="', "")
+            .replace('" title=""></span>', "")
+        )
+        return date
+    except Exception as e:
+        logging.error(f"It was not possible to obtain the date. Error: {e}")
+        return None
+
+
+def get_authors(response: HtmlResponse) -> list[str]:
+    return response.xpath(
+        '//p[contains(@class, "mb-0 text-muted")]//a//text()'
+    ).getall()
+
+
+def get_authors_links(response: HtmlResponse) -> list[str]:
+    return response.xpath('//*[contains(@class, "mb-0 text-muted")]//@href').get()
+
+
+def get_comments(response: HtmlResponse) -> list[dict[str, str]]:
+    return [None]
```

Comment:
Creo que en este caso es mejor retornar `[]` ya que esta representa mejor que no hay ningún comentario.
`[None]` podría interpretarse como una lista con un comentario vacío.

Además, aquí fallaría una comprobación de tipeo: para que caso que subiste, un tipeo correcto podría ser `list[dict[str, str] | None]`.

```suggestion
def get_comments(response: HtmlResponse) -> list[dict[str, str]]:
    return []
```

Reply:
fixed